### PR TITLE
#7 required optional

### LINKED ISSUE #7: Pass `required` as prop
Hi! Thanks for maintaining this.

Currently, the `textarea` always sets a `required` attribute. I think this should be optional, and not be on by default.

https://github.com/AdrianApan/yamde/blob/3ebb898db36e247958dd3931ed4e18f0ae0e9b42/src/Yamde.tsx#L113

### PR DESCRIPTION
**Pull request by joepio**
closes #7

### CODE REVIEW

File: src/Yamde.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const converter = new Showdown.Converter({
   noHeaderId: true,
 })
 
-const Yamde = ({ value, handler, toolbar }: Omit<Props, 'theme'>) => {
+const Yamde = ({ value, handler, toolbar, required }: Omit<Props, 'theme'>) => {
```
Comment (AdrianApan) [inline src/Yamde.tsx:18]:
```suggestion
const Yamde = ({ value, handler, toolbar, required = false }: Omit<Props, 'theme'>) => {
```

**Review by AdrianApan — COMMENTED** [review]
Hey, thank you for taking the time to contribute. I agree this would be a nice little addition. I left a suggestion, let me know what do you think. Furthermore I think it would also make sense to update the `README.md` file just to highlight the fact that this is an option now. Happy to merge and publish the new version afterwards of course. Cheers 🙌

**Comment by joepio** [comment]
Thanks for the suggestion, accepted it. Also updated readme. Maybe `autofocus` and some other HTML attributes should be added later, too.

**Comment by AdrianApan** [comment]
> Thanks for the suggestion, accepted it. Also updated readme. Maybe `autofocus` and some other HTML attributes should be added later, too.

Brilliant, thank you!

**Review by AdrianApan — APPROVED** [review]

**Comment by AdrianApan** [comment]
This is now published @joepio https://www.npmjs.com/package/yamde/v/1.7.0. Thank you for your contribution, much appreciated.

**Comment by joepio** [comment]
That was really fast, thank you!

**Comment by joepio** [comment]
Hmm, I installed 1.7.0, but it looks like the JS and d.ts files havent updated. Could that be true?

**Comment by AdrianApan** [comment]
> Hmm, I installed 1.7.0 (The `package.json` of the source in `node_modules` also verifies that), but it looks like the JS and d.ts files havent updated. Could that be true?

Thanks for flagging. I'll look into it now, will keep you in the loop.

**Comment by AdrianApan** [comment]
@joepio Can you give `1.7.1` a try? Seems like there was a hiccup with the dist folder. I've had a quick sanity check on a codesandbox env, seems ok now but I would mind a second confirmation on this. Thanks! 

CleanShot 2022-09-10 at 10 48 42@2x

**Comment by joepio** [comment]
1.7.1 works great, thanks! :D

**Pull request merged**